<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskWindow</class>
 <widget class="QDialog" name="TaskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TaskWindow {
	background-color: #ffffff;
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>18</y>
     <width>35</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>88</y>
     <width>299</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TE_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>300</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	background-color: #e9edfa;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #724cd7;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>300</width>
     <height>38</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: #e9edfa;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #724cd7;
	font-family:Impactl;
	font-size:12px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>301</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
      </property>
      <property name="text">
       <string>Deadline</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QDateEdit" name="DE_deadline">
        <property name="styleSheet">
         <string notr="true">QDateEdit {
    background-color: #e9edfa; 

    padding: 1px; 
	font-family: Calibri;
	font-size: 15px;
	color: #38446d;
	font-weight: bold;
}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_edit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #a196e7;
	border-radius:15px;
	display:inline-block;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:17px;
	font-weight:bold;
	padding:10px 45px;
	text-decoration:none;
	text-shadow:0px 1px 0px #2f6627;
}

QPushButton:hover {
	background-color: #b4abec;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
